File: fizzbuzz.js
const target = 200;

for (let count = 1; count < target; count++) {
	if (count % 3 == 0 && count % 5 == 0) {
		console.log('FizzBuzz!');
	} else if (count % 3 == 0) {
		console.log('Fizz!');
	} else if (count % 5 == 0) {
		console.log('Buzz!');
	} else {
		console.log(count);
	}
}

File: fizzbuzz.cpp
#include <iostream>

int main() {
  const int target = 200;

  for (int count = 1; count < target; count++) {
    if (count % 3 == 0 && count % 5 == 0) {
      std::cout << "FizzBuzz!" << std::endl;
    } else if (count % 3 == 0){
      std::cout << "Fizz!" << std::endl;
    } else if (count % 5 == 0){
      std::cout << "Buzz!" << std::endl;
    } else {
      std::cout << count << std::endl;
    }
  }
}
